CAM/PlaySynchronizedCamAnim.md

```diff
@@ -5,7 +5,7 @@ ns: CAM
 
 ```c
 // 0xE32EFE9AB4A9AA0C 0x9458459E
-BOOL PLAY_SYNCHRONIZED_CAM_ANIM(Any p0, Any p1, char* animName, char* animDictionary);
+BOOL PLAY_SYNCHRONIZED_CAM_ANIM(Camera camera,Scene scene, char* animName, char* animDictionary);
```

Comment:
`Camera` and `Scene` is not valid types. Use `Cam` and `int`. Also use `cs_type(Any)`, otherwise this change will break compatibility for C# ScRT.

Reply:
The `cs_type` annotation can be ignored when the [nativeType](https://github.com/citizenfx/fivem/blob/master/ext/natives/codegen_types.lua#L7-L8) and pointerness are the "same".

... Unless the argument types are either `func`, `Hash`, and `uint`, See: https://github.com/citizenfx/fivem/blob/master/ext/natives/codegen_out_cs.lua#L219-L224.